### PR TITLE
Fixes #821 - Fix SMR squawk colors

### LINKED ISSUE #821: Squawks on SMR profiles now grey/red
## What is the bug?
Unintentionally introduced by changing the TC_APP non concerned colour to grey to assist in de-cluttering approach displays, squawks now appear grey/red, whereas previously they were green/red. 

## How to reproduce?
Use any SMR profile.

## EuroScope Version
N/A

## Additional context
SMR symbology to be separated from TC_APP by doing the following:

1. Create Symbology_SMR.txt
2. point all SMR profiles' symbology settings line at created file
3. Setup colours appropriately
4. test

### PR DESCRIPTION
**Pull request by kristiankunc**
Fixes #821 

# Summary of changes

Created `Symbology_SMR.txt` from TC_APP symbology @ https://github.com/VATSIM-UK/uk-controller-pack/commit/2302aee82fb94652f5eedc681e89943478bcb8d3 and pointed all SMR profiles to it (except local symbology)

### CODE REVIEW

**Comment by kristiankunc** [comment]
Some colors may not be required in the file but I got no idea which ones.

**Comment by kristiankunc** [comment]
This already seems to do the trick but there might be unnecessary definitions, I've tested it on various SMR profiles and seems to work alright.

@BenWalker01 I know you joined the Discord discussion earlier so feel free to test it from my branch.

image

File: version.txt
Comment (luke11brown) [inline version.txt:1]: What's the purpose of this file?

Comment (kristiankunc) [reply version.txt:1]: included by accident, I've removed it now

**Review by luke11brown — APPROVED** [review]

**Pull request merged**